### DISCUSSION TITLE
Digital Signage Use?

### DISCUSSION THREAD

**Discussion by vanthome** [gerbera/gerbera#1547] [Q&A]
Hi, is it possible with the scripting feature to control the playlist based on external events. So let's say if a GPIOon a RaspberryPI triggers, the playback of a certain playlist item or a certain time offset?
BG

**Comment by whyman** [comment]
As Gerbera is a server this kind of control is not in its remit, you would need a UPNP-AV control point to emit the instruction, and Gerbera would simply provide the content.
